solutions/wallets/embedded-wallet/quickstart.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 
 title: "Quickstart"
-description: Quickstart guide for Embedded Wallet for the Sequence infrastructure stack for web3 gaming.
+description: Connect your users to your React App with the Sequence Embedded Wallet.

```

Comment:
This is not necessarily specific to React @VGabriel45 – Also can we put a mention of Unity and Unreal SDKs here too? In case people search for Embedded Wallet and just drop here.